docs/versioning/how-to-view-versions.md

```diff
@@ -0,0 +1,117 @@
+
+- Ensure you're logged in.
+- Navigate to the Registry.
+- Navigate to the item record.
+- Click on the "VERSIONING" tab as shown below.
+
+|                         Versioning Tab                                                                 |
+| :----------------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-registry-versioning-tab.jpg" alt="drawing" width="600"/> |
+
+
+You will be able to view the version history including the current and previous versions.
+
+|               Updated versioning details                                                    |
+| :----------------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-registry-create-version-result.png" alt="drawing" width="600"/> |
+
+You can click on the icon with 3 dots in the versioning timeline to expand to display additional versions.
+
+<img src="../assets/images/versioning/versioning-registry-view-expand.png" alt="drawing" width="400"/>
+
+Sorting is available in the drop down menu.
+
+<img src="../assets/images/versioning/versioning-registry-view-sort.png" alt="drawing" width="400"/>
+
+Clicking on the link of the ID of the version will enable you to navigate to that particular version.
+
+### Step 2. Viewing lineage
+
+If you would like to view the lineage graph of the item, click on the "EXPLORE LINEAGE" button to view the lineage.
+
+|             Select Explore Lineage button                                     |
+| :----------------------------------------------------------------------------------------------------: |
+|<img src="../assets/images/versioning/versioning-registry-view-lineage.png" alt="drawing" width="600"/> |
+
+You will see the direct lineage of the item in the provenance graph viewer as shown below.
+
+|              Explore Lineage in Provenance Store                                     |
+| :----------------------------------------------------------------------------------------------------: |
+|<img src="../assets/images/versioning/versioning-registry-explore-lineage.png" alt="drawing" width="600"/> |
+
+
+To view previous versions, you can use your mouse to double-click on the item node to expand other related nodes. 
+
+If you continue expanding the nodes including the version activity node, and respective items, you will be able to trace its version lineage in the provenance graph viewer as shown below.
+
+|             Explore version provenance lineage graph                                     |
+| :----------------------------------------------------------------------------------------------------: |
+|<img src="../assets/images/versioning/prov-store-version-lineage.gif" alt="drawing" width="600"/> |
+
+
+
+## Viewing version info in the Data Store
+
+### Step 1. Navigate to Versioning tab in the dataset record in the Data Store
```

Comment:
Do we need this section - maybe just explain that the user can navigate to the Data Store record in the data store, and click on the versioning tab. 

BUT the versioning is also fully functional in the registry for datasets as well - so they could just look at it in either. The view is identical.

Reply:
I debated about that. I thought I'd do both, as users may have different entry points to versioning. The Data Store view is slightly different - where the buttons are, where to find the lineage feature, etc.. that's all.